### DISCUSSION TITLE
Authentication on 2 instances

### DISCUSSION THREAD

**Discussion by aydar-s-v37** [datalust/seq-tickets#2209] [Help]
Hi! I have 2 seq instances running in the same linux host (from podman containers). Let's say one is http://my-seq/ and second is http://my-seq:8081. The problem is that being authenticated in one host logs me out from another. I understand it's because auth cookie is overwritten and because both seqs are on same "domain" it logs me out from the first. What would you suggest here? I need both to keep running, can't erase any data and can't erase any configuration - could it be something like setting up sub-domain for second one?

**Comment by nblumhardt** [comment]
HI! The best option here is to set up a different (sub) domain for each instance.

Seq does have a means to make authentication cookies unique in this scenario (the `--name` argument of `seqsvr run`), but the `run` invocation is tucked away in the container and customizing it would require building a customized container image, which requires more maintenance in the long run.

Hope this helps,
Nick

**Reply by aydar-s-v37** [reply]
Yeah that makes sense. Is it possible to have two docker/podman containers to use the same port 80? E.g. to have one instance on http://my-seq/ and second on http://sub.my-seq/ ?

**Reply by nblumhardt** [reply]
Putting an nginx container in front of the Seq containers can do this - https://serverfault.com/questions/579285/how-to-proxy-a-server-based-on-domain-name-of-request-using-nginx shows an example/has some discussion.
